style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+body {
+  background-image: url(../images/header-shapes-mobile.png);
+  background-size: 100%;
+  background-position: 0 80px;
+  background-repeat: no-repeat;
+}
+
```

Comment:
- [ ] Please make sure the background image is working correctly.

| Figma design | Your design |
|-------------------|------------------|
| ![Screenshot from 2022-11-05 20-43-06](https://user-images.githubusercontent.com/65129228/200133819-2778c2dc-7be7-452b-9636-4dbb935164a3.png) | ![Screenshot from 2022-11-05 20-35-54](https://user-images.githubusercontent.com/65129228/200133833-d82496db-663a-4cb9-a95b-e1d4031f39b1.png) |

- [ ] Make sure all the social media icons are present and correctly styled.